l10n_it_fatturapa_out_sp/tests/test_fatturapa_xml_validation.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2014 Davide Corio
+# Copyright 2015-2016 Lorenzo Battistini - Agile Business Group
+# Copyright 2018-2019 Alex Comba - Agile Business Group
+
+import base64
+
+from .fatturapa_common import FatturaPACommon
+
+
+class TestFatturaPAXMLValidation(FatturaPACommon):
+    def setUp(self):
+        super(TestFatturaPAXMLValidation, self).setUp()
+        self.company = self.env.company = self.sales_journal.company_id
```

Comment:
anche qui il setup viene già fatto in https://github.com/OCA/l10n-italy/blob/41cdfd0eddac063b6f0b4b58921eb4ad72f7f839/l10n_it_fatturapa_out/tests/test_fatturapa_xml_validation.py#L38-L64

Reply:
Si ma quella classe non la posso comunque ereditare senza portarmi dietro tutte le funzioni test_*, o sbaglio?

Reply:
> cosa intendi con?
> 
> > senza portarmi dietro tutte le funzioni test_*
> 
> Che vengono eseguiti i suoi test?

Si. Se non ricordo male la logica è che il test esegue tutti i metodi che iniziano per 'test_'.

Reply:
Ok, allora come detto in chiamata: lasciamo così in attesa di refactoring di tutti i test.